lgc/patch/llpcSystemValues.cpp

```diff
@@ -41,11 +41,12 @@
 using namespace lgc;
 using namespace llvm;
 
+// -enable-shadow-desc: enable shadow descriptor table
+static cl::opt<bool> EnableShadowDescriptorTable("enable-shadow-desc",
```

Comment:
enable-shadow-desc -> enable-shadow-desc-table

Reply:
The options are already used by the driver, so we can't change them now.

Reply:
Oh, I see. You have mentioned this in a previous change. I forgot.